src/applications/pact-act/containers/questions/TernaryRadios.jsx

```diff
@@ -74,36 +64,81 @@ const TernaryRadios = ({
 
   return (
     <>
-      <VaRadio
-        data-testid={testId}
-        onBlur={onBlurInput}
-        className="vads-u-margin-bottom--3"
-        error={(formError && 'Select a response.') || null}
-        hint=""
-        label={h1}
-        label-header-level="1"
-        onVaValueChange={onValueChange}
+      <div
+        className={
+          formError
+            ? 'vads-u-margin-bottom--3 form-question-error'
+            : 'vads-u-margin-bottom--3'
+        }
       >
-        {locationList}
-        <VaRadioOption
-          checked={formValue === responses[0]}
-          label={responses[0]}
-          name={shortName}
-          value={responses[0]}
-        />
-        <VaRadioOption
-          checked={formValue === responses[1]}
-          label={responses[1]}
-          name={shortName}
-          value={responses[1]}
-        />
-        <VaRadioOption
-          checked={formValue === responses[2]}
-          label={responses[2]}
-          name={shortName}
-          value={responses[2]}
-        />
-      </VaRadio>
+        <h1 className="form-question-header" id="form-question">
+          {h1}
+        </h1>
+        {locationList ? <div id="form-instructions">{locationList}</div> : null}
+        <fieldset
+          aria-labelledby="form-question form-instructions"
+          data-testid={testId}
+        >
+          {formError && (
+            <span className="usa-error-message" role="alert">
+              <div className="form-text-error">
+                <span className="usa-sr-only">Error</span> Select a response.
+              </div>
+            </span>
+          )}
+          <>
+            <input
+              type="radio"
+              checked={formValue === responses[0]}
+              data-testId="va-radio-option"
+              id={_.snakeCase(`${responses[0]}_input`)}
+              name={shortName}
+              onChange={() => onValueChange(responses[0])}
+              value={responses[0]}
+            />
+            <label
+              className="form-label"
+              htmlFor={_.snakeCase(`${responses[0]}_input`)}
+            >
+              <span>{responses[0]}</span>
+            </label>
+          </>
+          <>
+            <input
```

Comment:
Per the ticket description, aren't we able to continue using the radio buttons themselves (`<VaRadioOption>`) from the design system? I thought we only needed to replace the `<VaRadio>` and the bulleted list implementation.

If not, and we have to use raw radio buttons, see the below suggestion:

The cognitive load on this file increased quite a bit here, as well as code repetition. Maybe we could add a utility function like this:

```
const createRadioSet = response => {
  return (
    <>
      <input
        type="radio"
        checked={formValue === response}
        data-testid="va-radio-option"
        id={_.snakeCase(`${response}_input`)}
        name={shortName}
        onChange={() => onValueChange(response)}
        value={response}
      />
      <label
        className="form-label"
        htmlFor={_.snakeCase(`${response}_input`)}
      >
        <span>{response}</span>
      </label>
    </>
  );
};
```

Then you could map over the responses and call each one using this.

Reply:
Good catch, I was going to ask you about this when you came back. Since it wasn't implemented with a map at first I didn't know if it was going to be okay to do that with this implementation.

Reply:
Does this mean that we aren't going to keep using `<VaRadioOption>`?

Reply:
Ahh sorry for missing the first part of your question. I tried to remove just the VaRadio component, but this component takes in the onchange function. Unfortunately `va-radio-option` is not taking this in to use in its props in the DS component

Reply:
Gotcha. Ok, thanks!